### PR TITLE
Add responsive wrappers for tables 

### PR DESCRIPTION
**Pull request by rushenn**
And other minor UI fixes.

### CODE REVIEW

**Comment by djwfyi** [comment]
I still have the same issue.

image

**Review by ravindk89 — CHANGES_REQUESTED** [review]

File: source/_static/js/main.js [hunk 1]
```diff
@@ -334,4 +334,20 @@ window.addEventListener("DOMContentLoaded", (event) => {
       }
     }
   })();
+
+
+  // --------------------------------------------------
+  // Responsive tables
+  // --------------------------------------------------
+  (function () {
+    const tableEls = document.querySelectorAll("table");
```
[the remark this answers is not in the document]
Comment (rushenn) [reply source/_static/js/main.js:343]: I'm looking into this issue. 

By default, tables can not be set to overflow without a parent wrapping element. We'll have to wrap all the tables and style the parent to allow scrollbars incase overflowing content.

**Comment by rushenn** [comment]
@ravindk89 @djwfyi Sorry that I failed to commit the styles changes. I have update the code, PTAL.

**Comment by ravindk89** [comment]
That seems to have done it.

Minor thought - should the horizontal scrollbar be taller or otherwise more visible? Or is there some clever way of having the table look more clearly scrollable? I've always struggled with this. @oscarocastellanos in case you have design ideas.

**Comment by rushenn** [comment]
> That seems to have done it.
> 
> Minor thought - should the horizontal scrollbar be taller or otherwise more visible? Or is there some clever way of having the table look more clearly scrollable? I've always struggled with this. @oscarocastellanos in case you have design ideas.

@ravindk89 The current approach will use the native scrollbar, and it will defer from OS to OS. We can style the scrollbars with CSS to have more of a uniform look just like the current sidebars. I will work on this as part of my next PR.

**Comment by ravindk89** [comment]
Gotcha - @djwfyi if everything looks fixed on your end lets squash+merge this in

**Review by djwfyi — APPROVED** [review]
LGTM

**Pull request merged**